code/game/objects/items/storage/belt.dm

```diff
@@ -336,6 +336,13 @@
 	new /obj/item/reagent_container/spray/pepper(src)
 	new /obj/item/device/clue_scanner(src)
 
+/obj/item/storage/belt/security/MP/cadet/fill_preset_inventory()
+	new /obj/item/weapon/gun/energy/taser/safe(src)
```

Comment:
No need for cadets to be unable to use their tazer in this way, needless handholding.